ls1intum/Artemis
# Create Exam Scores Page

### Checklist

### Motivation and Context



Spring Backlog item

### Description

Added an exam scores board with csv export.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create an Exam, Conduct the Exam, Correct the Exam
4. Look at the results in the exam results page and check for errors / wrong calculations. ALso try the csv export

### Screenshots


![image](https://user-images.githubusercontent.com/29718932/86367003-c88e7500-bc7b-11ea-8ddd-c5d95cd8ce5d.png)
